webapp/packages/plugin-data-viewer/src/DatabaseDataModel/Actions/ResultSet/ResultSetDataContentAction.ts

```diff
@@ -42,6 +43,7 @@ export class ResultSetDataContentAction extends DatabaseDataAction<any, IDatabas
     private readonly graphQLService: GraphQLService,
     private readonly quotasService: QuotasService,
     private readonly cache: ResultSetCacheAction,
+    private readonly dataViewerService: DataViewerService,
```

Comment:
this is not responsibility of database data model - it presents only database not hight level API or logic